### PR TITLE
Maven: relocate platform-specific dependency

### PR DESCRIPTION
**Pull request by tool4ever**
Now GDX will be excluded from Desktop JAR.

### CODE REVIEW

File: forge-gui-desktop/pom.xml [hunk 1]
```diff
@@ -118,7 +118,7 @@
                         </goals>
                         <configuration>
                             <!-- TODO: insert placeholder for latest version tag -->
-                            <fromRef>forge-1.6.56</fromRef>
+                            <fromRef>forge-1.6.55</fromRef>
```
Comment (Hanmac) [inline forge-gui-desktop/pom.xml:121]: was this change on purpose?

Comment (tool4ever) [reply forge-gui-desktop/pom.xml:121]: yes, that's the latest tag we have available so it needs to start from there

**Review by kevlahnota — APPROVED** [review]
